### PR TITLE
Even more packages + Mail localization

### PR DESCRIPTION
**Pull request by Monotheonist**
## About the PR

Adds:
- Scarves and box mail that someone made a PR for and then instantly closed, so it never got in. Scarves has a small chance for syndicate scarves and a very very low chance for a CC scarf.
- Roboticist mail for Epistemics (ships 1 robot part and a battery.)

Tweaks:
- Moved the foam sabre into the general sword mail with the same probability as every other fake weapon in there.
- Localizes the frontier mail by making paper_mail.ftl and then doing a shitload of copy-paste action (the entire thing looks like hell, christ.)
- Tweaks comments in various files because there's just random spaces n' stuff.
## Why / Balance

It's just some extra stuff to add to the amazing bloat that the mail is, and also at the same time debloats it.
## Technical details

Technical changes are just YAML, random removal of space, reformatting of comments, and locstrings. If you want specifics look at the files changed.
## Media

Not necessary. It's pretty small.
## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Because of the way this PR is, pulling it to any other downstream may be a little fucky because it's two things (mail tweaks + new mail) and I'm going to apologize for that here, otherwise, sorry nothing.
Also the mail locale! _**IF IT BREAKS AND DOES NOT FORMAT IMMEDIATELY MAKE AN ISSUE AND I WILL TRY TO FIX IT.**_ Style tags in FTL files are fucky, apparently.

**Changelog**

:cl:
- add: Adds a few more parcels into the mail pool.

### CODE REVIEW

**Comment by Monotheonist** [comment]
Free from the clutches of the linter, finally, christ.

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml [hunk 1]
```diff
@@ -1732,12 +1790,3 @@
       amount: 1
     - id: PaperMailHydroCoTrial
       amount: 1
-
-- type: entity
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml]: undo this removal

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml (hunk 1 above)
Comment (Monotheonist) [reply Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml:1736]: WHOOPS!

Comment (Monotheonist) [reply Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml]: I'll do it, but I did mention it's also in the sword mail which is higher up in the file now. So like... ??? Double Sabre I guess.

**Pull request closed without merging**